### DISCUSSION TITLE
How to do an automatic export from node.js tiddlywiki to a single file?

### DISCUSSION THREAD

**Discussion by Ca1vino** [TiddlyWiki/TiddlyWiki5#5583] [Q&A]
I have seen how to do a manual export of selected tiddlers to a single file.

However, my use case is to be able to script and automate this as part of a daily scheduled backup process.  I would like to run a node program, or alternatively, use the tiddlywiki server command interface.

Is there a way to do this?  And if not, what would be the recommended way to add this capability?

**Comment by pmario** [comment]
see: https://tiddlywiki.com/#How%20to%20build%20a%20TiddlyWiki5%20from%20individual%20tiddlers:%5B%5BHow%20to%20build%20a%20TiddlyWiki5%20from%20individual%20tiddlers%5D%5D%20BuildCommand%20%5B%5Btiddlywiki.info%20Files%5D%5D

**Reply by Ca1vino** [reply]
Thanks, that answers the question perfectly. I might have seen that page but not really understood it properly.
